web/app/scripts/workspace/workspace-wrapper.js

```diff
@@ -237,6 +237,10 @@ angular.module('biggraph')
         if (!opts.willSaveLater) {
           this.saveWorkspace();
         }
+        const meta = this._boxCatalogMap[operationId];
+        // Collect usage statistics for placed boxes.
+        const loggedId = (meta && meta.categoryId !== 'Custom boxes') ? operationId : 'custom';
+        util.logUsage('box added', { operationId: loggedId });
```

Comment:
Maybe just don't send anything for custom boxes? Or change the document explaining the data collection. When I read there that "Custom boxes are excluded" I would have expected a behavior that they are not logged at all.

Reply:
> Maybe just don't send anything for custom boxes? Or change the document explaining the data collection. When I read there that "Custom boxes are excluded" I would have expected a behavior that they are not logged at all.

When I wrote that we didn't log anything for custom boxes. But then I noticed in the design doc we planned to log "custom box" for custom boxes. It makes sense — this way we see how often custom boxes are used.

If you agree, I'll update the policy page and keep the code like this.